README.md

```diff
@@ -77,6 +77,39 @@ For more information about the CLI and its arguments, run:
 $ python xain_fl/cli.py --help
 ```
 
+### Run the Coordinator from a Docker image
+
+Development image
+---
+
+To run the coordinator's development image, first build the Docker image:
+
+```shell
+$ docker build -t xain-fl-dev -f dev.dockerfile .
+```
+
+Then run the image, mounting the directory as a Docker volume, and call the
```

Comment:
Then run the image, mount the directory as a Docker volume, and call the

Reply:
"...and call the entrypoint". I added a line break to avoid long lines